admin/cron/mirror.sh

```diff
@@ -3,16 +3,16 @@
 MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../../" && pwd)
```

Comment:
This file will be referenced from people's crontabs, so we should probably include a symbolic link from slave.sh to mirror.sh in the repository.

Reply:
I *think* I did that right?